.github/workflows/osx.yml

```diff
@@ -15,20 +22,28 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
+
     - name: setup
       run: |
         brew install libomp
+
     - name: info
       run: |
         g++ -v
         cmake --version
+
+    - name: Debug over SSH (tmate)
+      uses: mxschmitt/action-tmate@v3.5
```

Comment:
Yes, it will continue to keep spitting the ssh connection link and will not continue to the next step. After you connect via ssh, all the previous steps will have been executed. When you exit the ssh connection, this step terminates. If you want to continue the job (move to the next step in the pipeline), then you do `sudo touch /continue` (create an empty file called `continue` in the root directory.), Then the ssh connection terminates and it moves to the next step.